### PR TITLE
ADD improved and new transformations

### PR DESCRIPTION
**Pull request by fgalan**
- [x] Step cjexl version
- [x] New time-related transformations (tests + doc)
- [x] CNR

### CODE REVIEW

File: doc/manuals/orion-api.md [hunk 1]
```diff
@@ -108,6 +108,9 @@
         - [`keys`](#keys)
         - [`arrSum`](#arrsum)
         - [`arrAvg`](#arravg)
+        - [`now`](#now)
+        - [`toIsoString`](#toisostring)
+        - [`getTime](#gettime)
```
Comment (fgalan) [inline doc/manuals/orion-api.md:113]:
```suggestion
        - [`getTime`](#gettime)
```

File: doc/manuals/orion-api.md [hunk 2]
```diff
@@ -3146,6 +3149,67 @@ results in
...
+#### now
+
+Returns the current time (plus a number of seconds specified as argument) as seconds since Unix epoch time.
+
+Extra arguments: none
+
+Example (being current time August 1st, 2024 at 9:31:02):
+
+```
+0|new
```
Comment (mapedraza) [inline doc/manuals/orion-api.md:3161]:
```suggestion
0|now
```

**Review by mapedraza — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
@fisuda this PR does some modifications to the English documentation. It would be great if you could sync the Japanese translation, please. Thanks!

**Comment by fisuda** [comment]
I sent the PR #4597.
Thanks.

**Pull request merged**